src/util/nodeSelectors.ts

```diff
@@ -190,6 +190,27 @@ export function selectWithLeadingDelimiter(...delimiters: string[]) {
   };
 }
 
+export function selectChildrenWithExceptions(excludedItems: string[] = []) {
```

Comment:
This is still WIP but I wanted to put it up in case there's an easier built-in way to address handling matching all children or matching all children with the exception of `x` type child or `y` type child.

Reply:
Also, docstring please 😊